core/src/test/resources/sql/winagg.iq

```diff
@@ -430,4 +430,29 @@ limit 5;
 
 !ok
 
+# NTH_VALUE
+select emp."ENAME", emp."DEPTNO",
+ nth_value(emp."DEPTNO", 1) over() as "first_value",
+ nth_value(emp."DEPTNO", 2) over() as "second_value",
+ nth_value(emp."DEPTNO", 5) over() as "fifth_value",
+ nth_value(emp."DEPTNO", 8) over() as "eighth_value",
+ nth_value(emp."DEPTNO", 10) over() as "tenth_value"
+from emp order by emp."ENAME";
```

Comment:
@snuyanzin , could you please clarify the test case?
Did you intentionally left `over()` part blank?
Did you know `nth_value` is computed last, so the order of rows in this query is undefined?

I'm inclined to add order by to `over` clauses, and use subquery to make result order predictable.

Reply:
It looks like I missed it while writing the tests.
Thanks for pointing to this.
I would say at least `order by emp."DEPTNO"` should be mentioned inside `over`.
I can push this change a bit later if there is no objections